src/emulator/functionsEmulatorRuntime.ts

```diff
@@ -100,6 +107,36 @@ interface ProxyTarget extends Object {
   obj.incremented == 2;
    */
 class Proxied<T extends ProxyTarget> {
+  /**
+   * Gets a property from the original object.
+   */
+  static getOriginal(target: any, key: string): any {
```

Comment:
The only change here is to make some methods `static` since they were not using the instance at all.